perun-base/src/main/resources/perun-roles.yml

```diff
@@ -2471,9 +2471,15 @@ perun_policies:
     include_policies:
       - default_policy
 
-  sponsorMember_Member_User_LocalDate_policy:
+  sponsored-sponsorMember_Member_User_LocalDate_policy:
```

Comment:
I do not think you want to split this policy into two. When you call method `sponsorMember` you don't have the SPONSORSHIP role with the member yet. therefore, what you want to use is this:
```
  sponsor-sponsorMember_Member_User_LocalDate_policy:
    policy_roles:
      - SPONSOR: Vo
        SELF: User
    include_policies:
      - default_policy

Reply:
Actually, it has to be split. But the policy is wrong.